### DISCUSSION TITLE
Did anyone tried to install APS on Shelter

### DISCUSSION THREAD

**Discussion by ghost** [zeapo/Android-Password-Store#2507] [Q&A] [answered]
I would like to try the version 2.0, however I don't really dear to install it as my main, since I don't want to break anything. I use [shelter](https://gitea.angry.im/PeterCxy/Shelter) to have a sandboxed workspace where I can install other applications and versions for test. I've tried multiple times with different version on my main workspace and my shelter workspace and it works. Did anyone tried that with APS? 

Bests,
Salus

**Comment by msfjarvis** [comment, marked as answer]
v2.0 has a different package name and will not upgrade your existing installation of APS v1.13.x, but instead be installed alongside it. You don't need a sandbox.

**Reply by ghost** [reply]
Will then create another .password-store file or will it write to the same as the v1.13.x?

**Reply by msfjarvis** [reply]
Each app has its own private data directory, neither app will be aware of or interfere with each other.

**Reply by ghost** [reply]
Thank you! I will give it a try :) Thanks for the amazing app by the way.

**Reply by msfjarvis** [reply]
Glad you like it.
